File: Meadow_DotNet_SDK/Project_Templates/templates/MeadowCoreComputeCSharp/MeadowApp.cs
using Meadow;
using Meadow.Devices;
using System.Threading.Tasks;

namespace MeadowApplication.Template;

public class MeadowApp : App<F7CoreComputeV2>
{
    public override Task Initialize()
    {
        Resolver.Log.Info("Initialize...");

        return Task.CompletedTask;
    }

    public override Task Run()
    {
        Resolver.Log.Info("Run...");

        Resolver.Log.Info("Hello, Meadow Core-Compute!");

        return Task.CompletedTask;
    }
}

File: Meadow_DotNet_SDK/Project_Templates/templates/MeadowCoreComputeFSharp/MeadowApp.fs
namespace MeadowApplication.Template

open Meadow
open Meadow.Devices
open System.Threading.Tasks

type MeadowApp() =
    inherit App<F7CoreComputeV2>()

    override this.Initialize() =
        do Resolver.Log.Info "Initialize..."

        Task.CompletedTask
        
    override this.Run () =
        do Resolver.Log.Info "Run..."

        do Resolver.Log.Info "Hello, Meadow Core-Compute!"

        Task.CompletedTask